src/main/resources/assets/bloodmagic/blockstates/BlockBloodTank.json

```diff
@@ -0,0 +1,92 @@
+{
+  "forge_marker": 1,
+  "defaults": {
+    "textures": { },
+    "model": "cube_all",
+    "uvlock": true
+  },
+  "variants": {
+    "tier": {
+	  "0": {
+		"textures": {
+	      "all": "bloodmagic:blocks/BloodTank"
```

Comment:
If you add 

```json
"textures": {
    "all": "bloodmagic:blocks/BloodTank"
}
```

to the `"default": {}` block, you shouldn't need it in all of these variants.